src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,204 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public Node<T> getHead() {
+        return head;
+    }
+
+    public void setHead(Node<T> head) {
+        this.head = head;
+    }
+
+    public Node<T> getTail() {
+        return tail;
+    }
+
+    public void setTail(Node<T> tail) {
+        this.tail = tail;
+    }
```

Comment:
you don't need these methods. cause you have access to the class variables inside the class. And you don't need to have an opportunity to change the value for head or tail from the outside, or to get them